Common/Data/Custom/Robintrack/RobintrackHoldings.cs

```diff
@@ -0,0 +1,164 @@
+        }
+
+        /// <summary>
+        /// Reads the data from the source provided in <see cref="GetSource(SubscriptionDataConfig, DateTime, bool)"/>
+        /// and converts it into an instance of this class (<see cref="RobintrackHoldings"/>).
+        /// </summary>
+        /// <param name="config">Subscription config</param>
+        /// <param name="line">Line to parse</param>
+        /// <param name="date">Date that the data is being read</param>
+        /// <param name="isLiveMode">Is live mode</param>
+        /// <returns>Instance of <see cref="RobintrackHoldings"/> casted as <see cref="BaseData"/></returns>
+        public override BaseData Reader(SubscriptionDataConfig config, string line, DateTime date, bool isLiveMode)
+        {
+            var instance = Read(line);
+            instance.Symbol = config.Symbol;
+
+            return instance;
+        }
+
+        /// <summary>
+        /// Creates a clone of the current object
+        /// </summary>
+        /// <returns>Copy of the current object</returns>
+        public override BaseData Clone()
+        {
+            return new RobintrackHoldings
+            {
+                EndTime = EndTime,
+                Symbol = Symbol,
+                Value = Value,
+                TotalUniqueHoldings = TotalUniqueHoldings,
+            };
+        }
+
+        /// <summary>
+        /// Parses a line of Robintrack data with an optional dateFormat
+        /// </summary>
+        /// <param name="line">Line to parse</param>
+        /// <param name="dateFormat">Date format to parse timestamp in</param>
+        /// <returns>Instance of this class</returns>
+        public static RobintrackHoldings Read(string line, string dateFormat = "yyyyMMdd HH:mm:ss")
+        {
+            var i = 0;
+            var csv = line.ToCsvData(size: 3);
+            var timestamp = Parse.DateTimeExact(csv[i++], dateFormat, DateTimeStyles.AdjustToUniversal);
+            var usersHolding = Parse.Decimal(csv[i++]);
+            var totalUniqueHoldings = Parse.Decimal(csv[i]);
+
+            return new RobintrackHoldings
+            {
+                EndTime = timestamp,
+                Value = usersHolding,
+                TotalUniqueHoldings = totalUniqueHoldings
+            };
+        }
+
+
+        /// <summary>
+        /// Sets the data timezone
+        /// </summary>
+        /// <returns>Timezone</returns>
+        public override DateTimeZone DataTimeZone()
+        {
+            return TimeZones.Utc;
+        }
+
+        /// <summary>
+        /// Enables mapping of the underlying symbol
+        /// </summary>
+        /// <returns>true</returns>
+        public override bool RequiresMapping()
+        {
+            return true;
+        }
+
+        /// <summary>
+        /// Provides list of supported resolutions
+        /// </summary>
+        /// <returns>List of supported resolutions</returns>
+        public override List<Resolution> SupportedResolutions()
```

Comment:
How often are these data points released? 
Maybe could keep a static list with the supported resolutions